chicken-sloths/bangazon-api-sprint1
# Joe create and fill tables

# Description
I used Tim's amazing function to create 4 tables: Computers, Employees, Departments, Training Programs
I also destructured the funcitons in the build-db.js file

## Number of Fixes


## Related Ticket(s)
Fixes #47 - Create Employees Table
Fixes #48 - Create Departments Table
Fixes #49 - Create Computers Table
Fixes #52 - Create Training Programs Table

## Problem to Solve
This creates 4 of the SQL database tables that we need

## Expected Behavior
if you run build-db.js it should create the 4 tables: Computers, Employees, Departments, Training Programs

## Steps to Test Solution

1. run 'node db/build-db.js'
2. check that the db was created INSIDE the db folder
3. open the database 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
